pyproject.toml

```
@@ -96,7 +96,7 @@ rich = ">=13.0.0"
 rich-click = "*"
 sqlalchemy = { version = ">=2.0.12", optional = true }
 structlog = { version = "*", optional = true }
-advanced-alchemy = { version = "0.2.1", optional = true }
+advanced-alchemy = { version = "0.2.2", optional = true }
```

```suggestion
advanced-alchemy = { version = ">=0.2.2", optional = true }
```

Maybe this would be better? @cofin